
https://bugzilla.redhat.com/show_bug.cgi?id=1050015

--- mod_auth_kerb-5.4/src/mod_auth_kerb.c.notyet
+++ mod_auth_kerb-5.4/src/mod_auth_kerb.c
@@ -1411,6 +1411,12 @@ verify_server_credentials(request_rec *r
                    "expire - now %d endtime %d",
                    princ_name, now, creds.times.endtime);
         *renew = 1;
+    } else if (now < creds.times.starttime) {
+        log_rerror(APLOG_MARK, APLOG_ERR, 0, r,
+                   "Credentials for %s are not yet valid. "
+                   "now %d starttime %d",
+                   princ_name, now, creds.times.starttime);
+        *renew = 1;
     } else {
         log_rerror(APLOG_MARK, APLOG_DEBUG, 0, r,
                    "Credentials for %s will expire at "
