
https://bugzilla.redhat.com/show_bug.cgi?id=970678

--- mod_auth_kerb-5.4/README.docs
+++ mod_auth_kerb-5.4/README
@@ -93,6 +93,14 @@ KrbDelegateBasic on | off       (set to
    authentication scheme in Apache (Apache 2.1 seems to provide better support
    for multiple various authentication mechanisms).
 
+KrbLocalUserMapping on | off    (set to off by default)
+   If set to 'on' this option, client principal names are mapped to local user
+   names in accordance with the local Kerberos configuration. 
+   For example, principal name "jim@EXAMPLE.COM" may be translated to "jim"
+
+KrbConstrainedDelegation on | off  (set to off by default)
+   If set to 'on', constrained delegation is enabled.  See below for more details.
+
 Note on server principals
 -------------------------
 Now you have to create an service key for the module, which is needed to
